### PR TITLE
Fix ilmerge issues

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Fixes issue : https://github.com/NuGet/Home/issues/3088

CC: @rrelyea @joelverhagen @rrelyea

### CODE REVIEW

**Comment by alpaix** [comment]
looks good. :shipit:

**Comment by joelverhagen** [comment]
:shipit:, let's clear the process with @rrelyea

**Comment by rohit21agrawal** [comment]
@alpaix have rebased against your recent changes..can you go through this once more quickly to see if you spot any issues?

File: build/common.ps1 [hunk 1]
```diff
@@ -640,6 +639,25 @@ Function Invoke-ILMerge {
...
+    $opts2 = , $nugetIntermediateExe
+    $opts2 += "/lib:$buildArtifactsFolder"
+    $opts2 += $nugetCore
+    if ($KeyFile) {
+        $opts2 += "/delaysign"
+        $opts2 += "/keyfile:$KeyFile"
+    }
+    $opts2 += "/out:$Artifacts\NuGet.exe"
+    if ($VerbosePreference) {
+    $opts2 += '/log'
```
Comment (alpaix) [inline build/common.ps1]: nit: fix indentation here..

File: build/common.ps1 [hunk 2]
```diff
@@ -640,6 +639,25 @@ Function Invoke-ILMerge {
...
+    }
+    Trace-Log "$ILMerge $opts2"
+    & $ILMerge $opts2 2>&1
+    if (-not $?) {
         Error-Log "ILMerge has failed. Code: $LASTEXITCODE"
     }
-}
+    Remove-Item $buildArtifactsFolder\$nugetIntermediateExe
+    Remove-Item $buildArtifactsFolder\$nugetIntermediatePdb
+    }
```
Comment (alpaix) [inline build/common.ps1]: ..and here

**Comment by alpaix** [comment]
looks good. :rocket:

**Pull request merged**